src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private static final int FIGURE_COUNT = 100;
+    private Random random = new Random();
+
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        int number = random.nextInt(FIGURE_COUNT);
+        String color = String.valueOf(ball.getRandomColor());
+        return color + number;
+    }
```

Comment:
`getRandomBall` method should return instance of `Ball`, not `String` (because of its name)
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants) (`String.valueOf` isn't so good)

also it would be better to have an appropriate constructor in the `Ball` class, 
so we could write something like `new Ball(color, number);`